demos/tests/run_tests.py

```diff
@@ -149,11 +149,14 @@ def main():
         dl_dir = prepare_models(auto_tools_dir, args.downloader_cache_dir, args.mo, global_temp_dir, demos_to_test)
 
         num_failures = 0
+        
+        demo_build_subdir = "/lib" if os.pathsep == ":" else ""
+        pythonpath_args = (os.environ['PYTHONPATH'], os.pathsep, args.demo_build_dir, demo_build_subdir)
+        PYTHONPATH = {'PYTHONPATH': "{}{}{}{}".format(*pythonpath_args)}
```

Comment:
This variable isn't really necessary; you could just embed the value inside the `demo_environment` initializer.

Also, using an f-string would be more readable than `str.format`.

Reply:
Done.